charts/keeper/values.yaml

```diff
@@ -0,0 +1,87 @@
+# -- Reservation configuration
+reservation:
+  # -- Whether reservation is enabled
+  enabled: true
+  # -- Reservation image configuration
+  image: registry.k8s.io/pause:3.9  # latest doesn't work on arm
+  # -- Reservation image pull policy
+  imagePullPolicy: Always
+  # -- Map of reservation deployments
+  map:
+#    default:
+#      replicas: 1
+#      nodeSelector: |
+#        node.kubernetes.io/instance-type: c5d.xlarge
+#      schedule:
+#        timeZone: Etc/UTC
+#        up:
+#          cron: "0 8 * * 1-5"
+#          replicas: 1
+#        down:
+#          cron: "0 17 * * 1-5"
+#          replicas: 0
+
+# -- Overprovisioning configuration
+overprovisioning:
+  # -- Whether overprovisioning is enabled
+  enabled: true
+  # -- Overprovisioning priorityClass configuration
+  priorityClass:
+    # -- Whether to create overprovisioning priorityClass
+    create: true
+    # -- Overprovisioning priorityClass priority
+    value: "-1000000"
+  # -- Overprovisioning priorityClass name override, will be used instead of priorityClass created in overprovisioning.priorityClass
+  priorityClassOverride: ""
```

Comment:
We could move this under `priorityClass` and rename to `name` and use if either for builtin priority class or custom one provided externally.